target/mips/translate_init.inc.c

```diff
@@ -522,6 +522,7 @@ const mips_def_t mips_defs[] =
         /* Note: Config1 is only used internally, the R4000 has only Config0. */
 #if defined(TARGET_CHERI)
         .CP0_Config1 = (1 << CP0C1_FP) | (47 << CP0C1_MMU) | (1 << CP0C1_C2),
+        .CP0_Config3 = MIPS_CONFIG3 | (1 << CP0C3_MT),
```

Comment:
Is this stale and something that should have been undone by 76859cb83606f913ddb1f1f4626bfd08dff8dfa9?

Reply:
Done